stereo calibration/CalibrationDataCollection.md

```diff
@@ -0,0 +1,69 @@
+# Collecting calibration data
+
+## Overview and rationale?
+...
+
+## Recommendations
+
+### 0. Arrangement of the cameras
+There are very nice recommendations on the arrangement of the cameras in the [StereoMorph docs](https://aaronolsen.github.io/tutorials/stereomorph/arranging_cameras_photography.html)
+- As much FOV overlap as possible? --- see StereoMorph
+- Remember! We are calibrating the cameras for a specific relative position of the cameras, zoom and focus levels!
+- Set the cameras to the smallest aperture
+
+### 1. Design of the checkerboard
+- The pattern should be on a flat and rigid surface 
+
+- White border around it, of roughly the same width as one of the cells (this makes the detection more robust in various environments see [here](https://docs.opencv.org/4.2.0/d9/d0c/group__calib3d.html#gad0e88e13cd3d410870a99927510d7f91))
+
+- Min number of **inner corners**
+    - *Why inner corners?*: "All chessboard related algorithm expects amount of inner corners as board width and height" [^2] - openCV parametrises the width and height based on number of inner corners
+    - 
+
+
+
+- Creating a calibration pattern
+    - [OpenCV](https://docs.opencv.org/4.x/da/d0d/tutorial_camera_calibration_pattern.html)
+    - [Calib.io](https://calib.io/pages/camera-calibration-pattern-generator)
+    - [Mark Hedley Jones blog post](https://markhedleyjones.com/projects/calibration-checkerboard-collection) --- not sure if these print out with a white border!
+
+### 2. Position and orientation of the checkerboard
+For a calibration frame to be useful, the checkerboard must fulfill the following criteria:
+
+- Keep the orientation of the checkboard within 90deg (conservatively 30 deg [^1]) of the "original"/default position
+    - *Why?* The algorithm takes the bottom (right?) corner of the board as the origin, and then traverses the rest of corners from there. If in some frames the board is rotated beyond 30 deg, the origin may be detected at a different physical corner, and all the corners may be traversd in a different order.
+
+- Cover several distances, and within each distance, cover (all parts of the image view)/(of the FOV overlap) [^1]
+    - *Why?*
+    - Note however that if at some distance the calibration pattern is too small, we may not be able to detect the corners reliably and the calibration may be poor. To fix this we may need a larger board (or move cameras closer to the area of interest?) 
+
+- Use a checkerboard as big as possible, ideally with of at least 8x6 squares and a square size of at least 100 mm [^1]
```

Comment:
Are we sure about 100mm? That's huge! 
If that's the case, to get 100mm 8x6 squares, we'd need 2x A2 checkerboards to fit together (over 2 meters in width!)

Reply:
yes I agree - we discussed this offline so marking as resolved